### PR TITLE
Change topics list to regex

### PR DESCRIPTION
**Pull request by mark-r-m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ and create tables in the `k2hb` namespace. The data will be stored in `cf:data`
 with at least `1` version and at most `10` versions and a TTL of 10 days.
 
 * **K2HB_HBASE_ZOOKEEPER_PARENT**
-    The hbase parant uri, defaults to `/hbase` but should be set to ``/hbase-unsecure`` for AWS HBase
+    The hbase parant uri, defaults to `/hbase` but should be set to `/hbase-unsecure` for AWS HBase
```
Comment (craigandrews) [inline README.md]: parant -> parent

Comment (craigandrews) [reply README.md]: I know it wasn't added in this change :D

File: Makefile [hunk 2]
```diff
@@ -29,7 +29,7 @@ destroy: down ## Bring down the Kafka2Hbase Docker container and services then d
 
 .PHONY: integration
 integration: up ## Run the integration tests in a Docker container
-	docker-compose run --rm integration-test ./gradlew integration
+	docker-compose run --rm integration-test ./gradlew --rerun-tasks integration
```
Comment (craigandrews) [inline Makefile]: presumably this isn't needed if the `upToDateWhen` thing is used below?

File: src/integration/kotlin/Kafka2Hbase.kt [hunk 3]
```diff
@@ -2,16 +2,16 @@ import io.kotlintest.shouldBe
...
 class Kafka2Hbase : StringSpec({
     configureLogging()
-    
-    val topic = "test-topic".toByteArray()
 
     val producer = KafkaProducer<ByteArray, ByteArray>(Config.Kafka.props)
     val hbase = HbaseClient.connect()
 
     "messages with new identifiers are written to hbase" {
+        val topic = "test-topic-%s".format(Instant.now().toEpochMilli()).toByteArray()
```
Comment (craigandrews) [inline src/integration/kotlin/Kafka2Hbase.kt]: maybe put this in a function alongside `uniqueBytes` and call it `uniqueTopic`

File: Makefile (hunk 2 above)
Comment (mark-r-m) [reply Makefile]: Done

File: src/integration/kotlin/Kafka2Hbase.kt (hunk 3 above)
Comment (mark-r-m) [reply src/integration/kotlin/Kafka2Hbase.kt]: Done

File: README.md (hunk 1 above)
Comment (mark-r-m) [reply README.md]: Done

File: src/main/kotlin/Config.kt [hunk 4]
```diff
@@ -51,9 +55,15 @@ object Config {
             put("value.serializer", ByteArraySerializer::class.java)
 
             put("auto.offset.reset", "earliest")
+            put(metaDataRefreshKey, getEnv("K2HB_KAFKA_META_REFRESH_MS") ?: "10000")
         }
 
-        val pollTimeout: Duration = getEnv("K2HB_KAFKA_POLL_TIMEOUT")?.toDuration() ?: Duration.ofDays(10)
-        val topics = getEnv("K2HB_KAFKA_TOPICS")?.split(',') ?: listOf("test-topic")
+        val pollTimeout: Duration = getEnv("K2HB_KAFKA_POLL_TIMEOUT")?.toDuration() ?: Duration.ofMinutes(10)
```
Comment (steveburton4) [inline src/main/kotlin/Config.kt]: This doesn't match the readme, which says:

_It will poll the `test-topic` topic with a poll timeout of 10 **days**, and refresh the topics list every 10 seconds_

Comment (mark-r-m) [reply src/main/kotlin/Config.kt]: Done

**Pull request merged**